knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -33,12 +33,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
-	kafkaconfig "knative.dev/eventing-kafka/pkg/common/config"
-	"sigs.k8s.io/yaml"
```

Comment:
@aliok  ah,

we got rid of this lib. Usually we should run `make generated-files` before submitting the PR, regadless if tests pass

Reply:
here is the fix for this, by the bot:

https://github.com/openshift-knative/serverless-operator/pull/1566